examples/local/launch_cloud_onprem.py

```diff
@@ -0,0 +1,81 @@
+import argparse
```

Comment:
Add a comment to top explaining what the script is and how to use it?

Reply:
Something like:
```
"""Launches an onprem cluster for testing purposes. 

<more details on how it works/ what it does. Also say something about how to clean up the created the cluster>

Usage:
  <usage example>
"""
```

Reply:
Done, thanks!